File: app/models/nutrition_fact.rb
# == Schema Information
#
# Table name: nutrition_facts
#
#  id          :integer          not null, primary key
#  dish_id     :integer
#  calories    :integer
#  fats        :integer
#  carbs       :integer
#  proteins    :integer
#  cholesterol :integer
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#

class NutritionFact < ActiveRecord::Base
  belongs_to :dish
  attr_accessible :calories, :carbs, :fats, :cholesterol, :proteins, :dish_id
end

File: public/tocarta/app/model/NutritionFact.js
/**
 * NutritionFact Class
 *
 */

Ext.define('TC.model.NutritionFact', {
	extend : 'Ext.data.Model',
	config: {
		fields: [
	    {name: "id", type: "int"},
	    {name: "dish_id", type: "int"},
	    {name: "description", type: "string"},
	    {name: "carbs", type: "int"},
	    {name: "fats", type: "int"},
	    {name: "proteins", type: "int"},
	    {name: "cholesterol", type: "int"},
	    {name: "calories", type: "int"}
	  ],
	  associations: [
	    {type: 'belongsTo', model: 'TC.model.Dish', name: 'dish'}
	  ]
	}
});
